tests/env/start-swift-ceph.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
![bildschirmfoto von 2017-08-10 16-52-37](https://user-images.githubusercontent.com/1005065/29176529-5bb7149e-7dec-11e7-90e0-c61817a04519.png)

why move to sudo: false? does this work together with service: docker? afaik not